src/NuGet.Services.AzureSearch/Owners2AzureSearch/IOwnerDataClient.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NuGetGallery;
+
+namespace NuGet.Services.AzureSearch.Owners2AzureSearch
+{
+    public interface IOwnerDataClient
+    {
+        Task<ResultAndAccessCondition<SortedDictionary<string, SortedSet<string>>>> ReadLatestIndexedAsync();
```

Comment:
Could you add comments about the purpose of the owners data and when the data is updated? Ideally someone could get a gist of today's design review from the comments on this class.

Reply:
Yeah, sounds good.

Reply:
Looks great! 👌